homeworks/03-data-structures/src/merge-sort.test.ts

```diff
@@ -0,0 +1,11 @@
+import {mergeSort} from "./merge-sort"
+
+/* - Develop the `mergeSort` function. */
+test('mergesort should sort', () => {
+    const randomArray = Array(100_000).fill(0).map(() => Math.random())
+    const compareFn = (a, b) => a - b
```

Comment:
The compare function parameters should have types defined instead of being `any`.